app/cdap/components/GridTable/components/Breadcrumb/styles.ts

```diff
@@ -21,22 +21,22 @@ export const useStyles = makeStyles({
   breadCombContainer: {
     display: 'flex',
     justifyContent: 'space-between',
-    height: '48px',
+    height: 48,
```

Comment:
Why no units on these dimensions?

Reply:
updated the code to reflect dimensions instead of relying on default dimensions.